drivers/pm_cpu_ops/Kconfig

```diff
@@ -0,0 +1,19 @@
+# CPU power management driver configuration options
+
+# Copyright (c) 2020 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config PM_CPU_OPS
+	bool "CPU Power Management Drivers"
+	help
+	  Enable CPU power management driver configuration
+
+if PM_CPU_OPS
+
+module = PM_CPU_OPS
+module-str = pm_cpu_ops
+source "subsys/logging/Kconfig.template.log_config"
+
+source "drivers/pm_cpu_ops/Kconfig.psci"
```

Comment:
something is still coupling with psci.  I think we should have global ops structure to register the ops. User / API need not worry about the underlying mechanism (psci or someother). For example RISC-V will register its SBI ops and someone may register local functions.

Reply:
What you think is coupling with PSCI? The `pm_cpu_ops` subsystem is now totally agnostic. The PSCI implementation is only using this agnostic interface. Here we are sourcing the PSCI Kconfig file.